helm/defectdojo/templates/media-pvc.yaml

```diff
@@ -1,22 +1,29 @@
 {{- $fullName := include "django.pvc_name" $ -}}
 {{ with .Values.django.mediaPersistentVolume }}
-{{- if and .enabled (eq .type "pvc") .persistentVolumeClaim.create   }}
+{{- if and .enabled (eq .type "pvc") .persistentVolumeClaim.create }}
 apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
+  {{- with .Values.extraAnnotations }}
+  annotations:
+    {{- range $key, $value := . }}
+    {{ $key }}: {{ quote $value }}
+    {{- end }}
+  {{- end }}
   labels:
     defectdojo.org/component: django
     app.kubernetes.io/name: {{ include "defectdojo.name" $ }}
     app.kubernetes.io/instance: {{ $.Release.Name }}
     app.kubernetes.io/managed-by: {{ $.Release.Service }}
     helm.sh/chart: {{ include "defectdojo.chart" $ }}
-    {{- with $.Values.extraLabels }}
-      {{- toYaml . | nindent 4 }}
+    {{- range $key, $value := .Values.extraLabels }}
+    {{ $key }}: {{ quote $value }}
```

Comment:
Is there any specific reason to use this syntax (not `toYaml`).
I'm not against it, just to understand. Are these personal preferences, or are there any other benefits?